examples/multiphase_flow/ex13/LevelSetSolidInitialCondition.h

```diff
@@ -62,7 +62,7 @@ class LevelSetSolidInitialCondition : public CartGridFunction
                         Pointer<Patch<NDIM> > patch,
                         const double data_time,
                         const bool initial_time = false,
-                        Pointer<PatchLevel<NDIM> > patch_level = Pointer<PatchLevel<NDIM> >(NULL));
+                        Pointer<PatchLevel<NDIM> > patch_level = Pointer<PatchLevel<NDIM> >(nullptr));
```

Comment:
we're a little inconsistent on these (we don't have to specify the type on the right) but it isn't a big deal.

Reply:
Let's go ahead and fix those too; see #1736.